src/test/scala/org/apache/spark/sql/delta/DescribeDeltaHistorySuite.scala

```diff
@@ -598,6 +615,8 @@ trait DescribeDeltaHistorySuiteBase
         )
         checkOperationMetrics(
           expectedMetrics, operationMetrics, DeltaOperationMetrics.DELETE_PARTITIONS)
+        assert(operationMetrics("scanTimeMs").toLong > 0)
```

Comment:
Could we perhaps move this in the checkOperationMetrics

Reply:
sure, I'll try. sorry for late reply